pix-editor/app/components/field/tutorials.js

```diff
@@ -74,23 +75,33 @@ export default class Tutorials extends Component {
 
   @action
   closeTutorialPopin() {
+    this.tutorial.rollbackAttributes();
     this.displayTutorialPopin = false;
   }
 
   @action
   async saveTutorial() {
-    this.displayTutorialPopin = false;
     this.loader.start();
     try {
-      const tutorial = await this.tutorial.save();
+      if (this.tutorial.link) {
+        this.tutorial.link = this.tutorial.link.replaceAll(' ', '');
+        new URL(this.tutorial.link);
+      }
+    } catch {
       this.loader.stop();
+      this.notify.error('Lien du tutoriel non valide');
+      return;
+    }
```

Comment:
Ca marche, mais du coup, j'ai l'impression qu'on passera plus jamais dans le catch. Ptêt que le try catch n'a plus d'intérêt. A moins qu'il y ait d'autres façons de mal formater un lien de tuto ?

```suggestion
    } catch {
      this.loader.stop();
      this.notify.error('Lien du tutoriel non valide');
      return;
    }
```